docs/dev/compiler-toolchain/solidity.md

```diff
@@ -1,33 +1,32 @@
 # Solidity
 
 The compiler we provide as a part of our toolchain is called [zksolc](https://github.com/matter-labs/zksolc-bin). It
-operates on IRs and metadata received from the underlying [solc](https://docs.soliditylang.org/en/latest/) compiler,
-which must be available in `$PATH`, or the path must be explicitly passed via the CLI (command-line interface).
+operates on IR and metadata received from the underlying [solc](https://docs.soliditylang.org/en/latest/) compiler,
+which must be available in `$PATH`, or its path must be explicitly passed via the CLI (command-line interface).
 
 ## Usage
 
-Most of the time, using our compiler via the Hardhat plugin is enough, and you do not have to dive deeper into its
-interface and I/O (input/output) methods. However, knowledge of these can be crucial for integration, debugging, or contribution purposes.
+Using our compiler via the Hardhat plugin usually suffices. However, knowledge of its interface and I/O (input/output)
+methods are crucial for integration, debugging, or contribution purposes.
 
 The CLI supports several I/O modes:
 
 1. Standard JSON.
 2. Combined JSON.
 3. Free-form output.
 
-All three modes use the standard JSON `solc` interface internally which helps to reduce the complexity of the `zksolc`
-interface and facilitate testing.
+All three modes use the standard JSON `solc` interface internally. This reduces the complexity of the `zksolc`
```

Comment:
Why not `it reduces`?

Reply:
If you say `it reduces`, `it` can refer to singular specific things in the previous sentence, i.e. three modes, or solc. `This` unequivocally refers to the whole process described by previous sentence.

Reply:
Thanks!